src/clims/models/transition.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+from django.db import models
+from sentry.db.models import (Model, FlexibleForeignKey, sane_repr)
+
+
+class TransitionType(object):
+    MOVE = 0
+    SPAWN = 1
+
+
+class Transition(Model):
+    """
+    Represents a Transition for a Substance.
+    """
+    __core__ = True
+
+    BACKEND_CAMUNDA = 'camunda'
+
+    # Added because it seems to be used in all CLIMS models.
+    # TODO: check if it is actually needed?
+    organization = FlexibleForeignKey('sentry.Organization')
+
+    # Added in case we want to be able to audit a user's history directly.
+    user = FlexibleForeignKey('sentry.User', related_name='transitions')
+
+    # The work batch in which this transition was created, if it was created in a work batch context.
+    # TODO: should we disallow nulls here? Are transitions ever created outside a work batch context?
+    work_batch = FlexibleForeignKey('WorkBatch', null=True, related_name='transitions')
+
+    source_location = FlexibleForeignKey('SubstanceLocation', related_name='source_transitions')
+    target_location = FlexibleForeignKey('SubstanceLocation', related_name='target_transitions')
+    source_substance = FlexibleForeignKey('Substance', related_name='source_transitions')
```

Comment:
I think only the source_location and target_location are really needed here, as they include a link to the substance (and version) in it.

So you can think of `SubstanceLocation` as a pointer to a substance, it knows where it resides.

Reply:
That's useful feedback, thanks. Removing.